test/Fail/Issue5448-4-noK.err

```diff
@@ -0,0 +1,10 @@
+Issue5448-4-noK.agda:8,1-15
+This clause has target type P x which is not usable at the required
+modality @ω
+
+Note: when --without-K is enabled, the target type must be usable
+at the modality in which the function was defined, since it is used
+to compute substitutions in Cubical Agda.
```

Comment:
I think we should avoid mentioning Cubical Agda in the error message for `--without-K`. When `--without-K` is used we do not actually "compute substitutions in Cubical Agda", so the text might be misleading. Note that I'm currently not aware of anything that would go wrong if we removed this restriction for `--without-K`. However, the proof sketch that we use to motivate the current implementation suggests something like this restriction, and the proof sketch is for a language with cubical features.

By the way, what does "the modality in which the function was defined" mean?

Reply:
It's the attribute attached to the function's name:
```agda
@0 subst : (A : Set) (@0 : A → Set) {x y : A} → x ≡ y → P x → P y
subst ... = {! P x has to be usable @0 !}
```

Anyway, I can't think of a good wording for the `Note:` in the `--without-K` case, and I don't want to skip the check entirely because it contradicts "`--cubical-compatible` = `--without-K` + internal cubical code generation", since `--cubical-compatible` would (by design, rather than by a bug) reject more programs than `--without-K`

Reply:
How about `because in univalent type theory, substitution is computed by cases on the predicate`?

Reply:
> How about `because in univalent type theory, substitution is computed by cases on the predicate`?

That may or may not be true, depending on how "univalent type theory" is defined.

Perhaps we can just skip (parts of) the explanation when `--without-K` is used.

Reply:
Works for me.